tests/E2E/FileTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @copyright 2015-2017 Contentful GmbH
+ * @license   MIT
+ */
+
+namespace Contentful\Tests\E2E;
+
+use Contentful\Delivery\Client;
+use Contentful\File\File;
+use Contentful\File\ImageFile;
+use Contentful\File\UploadFile;
+
+class FileTest extends \PHPUnit_Framework_TestCase
```

Comment:
As we just discussed: Split this up by putting the regular file/image file tests into AssetBasicTest (and using `cfexampleapi` - maybe just extend the existing tests) and rename this one to something like `InvalidDataInPreviewTest` or something like that.